ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/vllm/426__model_serving_vllm_metrics.robot

```diff
@@ -0,0 +1,119 @@
+*** Settings ***
+Documentation     Basic vLLM deploy test to validate metrics being correctly exposed in OpenShift
+Resource          ../../../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../../../Resources/OCP.resource
+Resource          ../../../../../Resources/Page/Operators/ISVs.resource
+Resource          ../../../../../Resources/Page/ODH/ODHDashboard/ODHDashboardAPI.resource
+Resource          ../../../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource          ../../../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../../../Resources/CLI/ModelServing/llm.resource
+Resource          ../../../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource
+Library           OpenShiftLibrary
+Suite Setup       Suite Setup
+Suite Teardown    Suite Teardown
+Test Tags         KServe
+
+
+*** Variables ***
+${VLLM_RESOURCES_DIRPATH}=    ods_ci/tests/Resources/Files/llm/vllm
+${DL_POD_FILEPATH}=           ${VLLM_RESOURCES_DIRPATH}/download_model.yaml
+${SR_FILEPATH}=               ${VLLM_RESOURCES_DIRPATH}/vllm_servingruntime.yaml
+${IS_FILEPATH}=               ${VLLM_RESOURCES_DIRPATH}/vllm-gpt2_inferenceservice.yaml
+${INFERENCE_INPUT}=           @${VLLM_RESOURCES_DIRPATH}/query.json
+${INFERENCE_URL}=             http://localhost:8080/v1/chat/completions
+${METRICS_URL}=               http://localhost:8080/metrics/
+${TEST_NS}=                   vllm-gpt2
+@{SEARCH_METRICS}=            vllm:cache_config_info
```

Comment:
Just curious, where did you get this metrics list and is it full list?

Reply:
https://docs.vllm.ai/en/latest/serving/metrics.html ..also we should get the list when we <inf_url>/metrics url

Reply:
I got them from the /metrics/ endpoint, but they appear to match the ones in Tarun's link